plugins/aws_sts/app/views/samson_aws_sts/_fields.html.erb

```diff
@@ -0,0 +1,45 @@
+<fieldset>
+  <legend>AWS STS Tokens</legend>
+  <div class="col-lg-offset-2">
+    <div class="form-group">
+      <div class="col-lg-6">
+        <p>Tokens will be requested using the following user: <%= SamsonAwsSts::Client.new.caller_user_id %></p>
```

Comment:
good chance that this can fail ... needs a `rescue "error connecting to aws, check SAMSON_STS_AWS env vars"` ?